ownCloudAppShared/Client/User Interface/ClientItemCell.swift

```diff
@@ -19,6 +19,29 @@
 import UIKit
 import ownCloudSDK
 
+extension NSMutableAttributedString {
+	var boldFont:UIFont { return UIFont.systemFont(ofSize: 17.0, weight: .semibold) }
+	var normalFont:UIFont { return UIFont.systemFont(ofSize: 15.0)}
+
+	func bold(_ value:String) -> NSMutableAttributedString {
```

Comment:
@hosy I think naming of these methods is a bit misleading. I would rather explicitly call them `appendBold(`) and `appendNormal() `respectively.

Reply:
renamed both